dbfit-java/core/src/main/java/dbfit/api/DdlStatement.java

```diff
@@ -1,13 +1,13 @@
-package dbfit.util;
+package dbfit.api;
 
 import java.sql.Statement;
 import java.sql.SQLException;
 
-public class DdlStatementExecution implements AutoCloseable {
+public class DdlStatement implements AutoCloseable {
```

Comment:
I don't know, `DdlStatement` I think is not so bad. It doesn't even extend `DbStatement`.